components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -44,15 +54,8 @@ export default function TeamSettings() {
         document.location.href = gitpodHostUrl.asDashboard().toString();
     };
 
-    const settingsMenu = [
-        {
-            title: 'General',
-            link: [`/t/${team?.slug}/settings`]
-        }
-    ]
-
     return <>
-        <PageWithSubMenu subMenu={settingsMenu} title='General' subtitle='Manage general team settings.'>
+        <PageWithSubMenu subMenu={getTeamSettingsMenu(team)} title="Settings" subtitle="Manage general team settings.">
```

Comment:
**suggestion**: What do you think of :a: renaming this menu _General_ to follow the pattern we use for team settings and :b: reordering this so it's always the first sub menu entry?

Reply:
🅱️: Sure 👍 happy to make this the top-level entry in the left submenu.

🅰️: I actually find it confusing when the BIG TITLE at the top of the page is different from the Tab I just clicked on: When I click on "Settings", I expect to see "Settings" (and not "General" -- I don't know what that is yet, so I think I mis-clicked).

So I had actually renamed the other "General" title in this PR 🙈

| Project Settings | Team Settings |
| --- | --- |
| <img width="1437" alt="Screenshot 2021-12-13 at 15 58 45" src="https://user-images.githubusercontent.com/599268/145835979-75166548-28b5-496f-a5b4-41a1cde60f8a.png"> | <img width="1436" alt="Screenshot 2021-12-13 at 15 58 10" src="https://user-images.githubusercontent.com/599268/145836096-c0f07bdc-2cc7-48c1-b053-e829a07c9076.png"> |

You'll notice that I've also renamed the left submenu entry to `Project` and `Team` respectively, because I felt that made sense. However, I'm happy to rename both of these to `General` (but keep `Settings` as the BIG TITLE at the top).

What do you think? 🏓

Reply:
:b: DEAL 🤝 

:a: Valid concern! This pattern was inherited for team settings from the personal account settings where is makes more sense to differentiate multiple options, see [`/settings`](https://gitpod.io/settings). Keeping the _Settings_ title could make sense for now that there're only two options but could be an issue when project and team settings grow over time. For example, see https://github.com/gitpod-io/gitpod/issues/4456, https://github.com/gitpod-io/gitpod/issues/5067, https://github.com/gitpod-io/gitpod/issues/7101, https://github.com/gitpod-io/gitpod/issues/6706#issuecomment-970649837, etc.